src/components/map/map.module.css

```diff
@@ -1,4 +1,117 @@
-.mapContainer {
+.mapWithHeader {
   height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+.header {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-medium);
+  background: var(--static-background-background_0-background);
+  color: var(--static-background-background_0-text);
+  border-radius: var(--border-radius-regular) var(--border-radius-regular) 0 0;
+  padding: var(--spacings-medium);
+}
+
+.header__icons {
+  display: flex;
+  gap: var(--spacings-medium);
+}
+
+.header__icons div {
+  background-color: var(--static-background-background_accent_0-background);
+  color: var(--static-background-background_accent_0-text);
+  padding: 6px;
   border-radius: var(--border-radius-regular);
+  width: 40px;
+  height: 40px;
+}
+
+.header__info {
+  display: flex;
+  flex-direction: column;
+}
+
+.header__info__secondary {
+  color: var(--text-colors-secondary);
+}
+
+.flexOrder__2 {
+  order: 2;
+}
+
+.header__buttons {
+  display: flex;
+  gap: var(--spacings-medium);
+  margin-left: auto;
+}
+
+.mapContainer {
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.header__leftContainer {
+  display: flex;
+  gap: var(--spacings-medium);
+}
+
+.map__buttons {
+  position: absolute;
+  bottom: var(--spacings-xLarge);
+  right: var(--spacings-medium);
+  z-index: 1;
+}
+
+.map__buttons button {
+  margin: var(--spacings-medium);
+}
+
+.map__closeButton {
+  display: none;
+}
+
+.map__showButton {
+  display: none !important;
```

Comment:
Is it not possible to get away without using `!important`?

Reply:
It probably is, but I didn't find out why. I'll look into it when I continue refactoring 👍

Reply:
Fixed 👍